tests/test_otmm_makam.py

```diff
@@ -0,0 +1,707 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from mirdata import annotations
+from mirdata.datasets import compmusic_otmm_makam
+from tests.test_utils import run_track_tests
+
+
+def test_track():
```

Comment:
Echoing on @rabitt's [comment](https://github.com/mir-dataset-loaders/mirdata/pull/435#r566016550_) regarding the length of unit tests. It's a bit difficult to capture what each test at a glimpse due to their size. 

The ideal approach might be to use [mocks](https://docs.python.org/3/library/unittest.mock.html) to decouple the functionality you are testing from the data. On the other hand, I appreciate that mocks are a bit difficult to get your head around. Please disregard my suggestion if you think it's overkill.